addon/lib/connect.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+import reduxStore from '../services/redux-store';
+
+const {
+  computed,
+  defineProperty,
+  inject: { service }
+} = Ember;
+
+export default function connect(mapStateToComputed, mapDispatchToActions, storeService) {
+  return function wrapWithConnect(WrappedComponent) {
+    return WrappedComponent.extend({
+      reduxStore: storeService || service('reduxStore'),
+
+      init() {
+        this._super(...arguments);
+        this.actions = this.actions || {};
+        const propMap = Ember.typeOf(mapStateToComputed) === 'function' ? 
+          mapStateToComputed(reduxStore) :
```

Comment:
p0: I think you meant to pass in get(this, 'reduxStore'); -> currently, you are passing in an uninjected service function. Not sure if that was intentional, but it makes the argument passed into stateMap and actionMap less useful :)